### PR TITLE
Reyhane 912

### PR DESCRIPTION
**Pull request by reyhane-rangamiz**
a comprehensive article for both development and production environment setup has been composed.

### CODE REVIEW

**Review by ShammasAmir — CHANGES_REQUESTED** [review]
Hi. Some minor changes are requested.

File: 8.x/getting-started/installation.md [hunk 1]
```diff
@@ -269,148 +472,344 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+ |Preference : | 0|
+ |Content type :| regular|
+ |Enabling web page| on |
+
+ Save changes!
+
+Afterward, another form will be displayed. Here you may need to build a theme file prior to editing the web page.
+To do so, follow this step-by-step instruction:
+
+1. Create a file on `/larammerce-base-theme/public/views/$`.
```
[the remark this answers is not in the document]
Comment (reyhane-rangamiz) [reply 8.x/getting-started/installation.md:778]: @ShammasAmir 
Thank you for the attention. it was a writing mistake. completely solved.

File: 8.x/getting-started/installation.md [hunk 2]
```diff
@@ -269,152 +472,332 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
-Add the following to the file named `.envrc`:
+export ECOMMERCE_BASE_PATH=/home/your-sysytem-name/your-root/larammerce
+```
+Note: On the main directory run `pwd` to get the path used in the command above.
 
-```bash
-export ECOMMERCE_BASE_PATH=/var/www/larammerce 
+Here, you may run to an access issue, to solve it run:
+```
+direnv allow
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:727]: Add "space" and "dot" -> direnv allow .

**Review by ShammasAmir — CHANGES_REQUESTED** [review]
@reyhane-rangamiz : Hi. One minor change proposed.

**Review by ShammasAmir — CHANGES_REQUESTED** [review]
Hi. The document is great. Some changes have been suggested.

**Review by ShammasAmir — CHANGES_REQUESTED** [review]
Hi. The document is great. Some changes have been suggested.

File: 8.x/getting-started/installation.md [hunk 3]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
 ```bash
 ./deploy.sh
 ```
 
+---
+
+#### Working with the theme
+
+As a sample, lets build a web page! 
+
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:834]: In the browser, type "http://0.0.0.0:8080/admin" to see the login page.
In the terminal, go to the `~/projects/larammerce/database/seeders/` directory and run:
```bash
vim UsersTableSeeder.php
```
In this file, you can see the username and password for admin user. Insert these values into the login fields to enter the admin panel.

File: 8.x/getting-started/installation.md [hunk 4]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+Once you logged in as admin on `localhost:8080/admin`, press plus button on the left buttom of the page ,create a folder and equalized some random values in the form displayed.
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:835]: `localhost:8080/admin` -> `http://0.0.0.0:8080/admin`
equalized -> enter

File: 8.x/getting-started/installation.md [hunk 5]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+2.Add the script below to the file:
+
+```
+@php
+
+echo "hello world!"
+
+@endphp
+```
+3.Now deploy the project on `~ /larammerce-base-theme` directory:
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:866]: `~ /larammerce-base-theme` -> `larammerce-base-theme`

File: 8.x/getting-started/installation.md [hunk 6]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+```
+./deploy.sh 
+```
+
+Finally, press `edit web page` button in the form, select the url name you created, and save.(here the url name is test)<br>
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:871]: select the url name -> select the blade file name

here the url name is test -> Here the blade file name is `test`

File: 8.x/getting-started/installation.md [hunk 7]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+ |------------|-------|
+ |Title: | Test folder|
+ |url part: | test |
+ |Preference : | 0|
+ |Content type :| regular|
+ |Enabling web page| on |
+
+ Save changes!
+
+Afterward, another form will be displayed. Here you may need to build a theme file prior to editing the web page.
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:849]: build a theme file -> build a blade file

File: 8.x/getting-started/installation.md [hunk 8]
```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
...
+
+@endphp
+```
+3.Now deploy the project on `~ /larammerce-base-theme` directory:
+```
+./deploy.sh 
+```
+
+Finally, press `edit web page` button in the form, select the url name you created, and save.(here the url name is test)<br> 
+See the result on `localhost:8080/test`.
```
Comment (ShammasAmir) [inline 8.x/getting-started/installation.md:872]: `localhost:8080/test` -> ` http://0.0.0.0:8080/test`

**Review by ShammasAmir — CHANGES_REQUESTED** [review]
Some other changes have been proposed.

**Pull request merged**